pubsub/subscriptions.rb

```diff
@@ -0,0 +1,253 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "google/cloud/pubsub"
+
+def update_push_configuration project_id:, subscription_name:, new_endpoint:
+  # [START update_push_configuration]
+  # project_id        = Your Google Cloud Project ID
```

Comment:
Add double quotes around value. 
For example:
`project_id = "Your Google Cloud Project ID"`